core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -106,6 +131,62 @@ public void testImportMutationsFile() throws Exception {
+        assertNull(DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations"));
+    }
+
+    @Test
+    public void testImportSplitMutationsFile() throws Exception {
+        /* Mutations file split over two files with same stable id */
+        String[] args = {
+                "--data","src/test/resources/splitMutationsData/data_mutations_extended.txt",
+                "--meta","src/test/resources/splitMutationsData/meta_mutations_extended.txt",
+                "--loadMode", "bulkLoad"
+        };
+        ImportProfileData runner = new ImportProfileData(args);
+        runner.run();
+        String studyStableId = "study_tcga_pub";
+        CancerStudy study = DaoCancerStudy.getCancerStudyByStableId(studyStableId);
+        studyId = study.getInternalId();
+        int sampleId = DaoSample.getSampleByCancerStudyAndSampleId(studyId, "TCGA-AA-3664-01").getInternalId();
+        int secondSampleId = DaoSample.getSampleByCancerStudyAndSampleId(studyId, "TCGA-AA-3665-01").getInternalId();
+        GeneticProfile geneticProfile = DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations");
+        geneticProfileId = geneticProfile.getGeneticProfileId();
+
+        // assume clinical data for MUTATION_COUNT was created
+        ClinicalAttribute clinicalAttribute = DaoClinicalAttributeMeta.getDatum("MUTATION_COUNT", studyId);
+        assertNotNull(clinicalAttribute);
+
+        // assume a MUTATION_COUNT record has been added for both samples
+        List<ClinicalData> clinicalData = DaoClinicalData.getSampleData(study.getInternalId(), new ArrayList<String>(Arrays.asList("TCGA-AA-3664-01", "TCGA-AA-3665-01")), clinicalAttribute);
+        assert(clinicalData.size() == 2);
+        assertEquals("3", clinicalData.get(0).getAttrVal());
+        assertEquals("0", clinicalData.get(1).getAttrVal());
+
+        // load a second mutation data file
+        String[] secondArgs = {
+                "--data","src/test/resources/splitMutationsData/data_mutations_extended_continued.txt",
+                "--meta","src/test/resources/splitMutationsData/meta_mutations_extended.txt",
+                "--loadMode", "bulkLoad"
+        };
+        ImportProfileData secondRunner = new ImportProfileData(secondArgs);
+        secondRunner.run();
+        // assume a MUTATION_COUNT record has been updated for both samples (both +1)
+        clinicalData = DaoClinicalData.getSampleData(study.getInternalId(), new ArrayList<String>(Arrays.asList("TCGA-AA-3664-01", "TCGA-AA-3665-01")), clinicalAttribute);
+        assert(clinicalData.size() == 2);
+        assertEquals("4", clinicalData.get(0).getAttrVal());
+        assertEquals("1", clinicalData.get(1).getAttrVal());
+        // remove profile at the end
+        DaoGeneticProfile.deleteGeneticProfile(geneticProfile);
+        assertNull(DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations"));
```

Comment:
this unit test is not checking whether the deleteGeneticProfile() works properly, so we may not need this assert